src/components/organisms/DropdownContent/DropdownContent.tsx

```diff
@@ -0,0 +1,151 @@
+import { Dispatch, FC, SetStateAction, useState } from 'react'
+import classNames from 'classnames'
+import { includes, map } from 'lodash'
+import CheckBoxInput from 'components/molecules/CheckBoxInput/CheckBoxInput'
+import Button, {
+  AppearanceTypes,
+  SizeTypes,
+} from 'components/molecules/Button/Button'
+import { SelectionControlsInputProps } from 'components/organisms/SelectionControlsInput/SelectionControlsInput'
+
+import classes from './styles.module.scss'
+
+type DropdownContentProps = SelectionControlsInputProps & {
+  isOpen?: boolean
+  selectedOptionObjects?: { label: string; value: string }[]
+  setIsOpen?: Dispatch<SetStateAction<boolean>>
+}
+
+const DropdownContent: FC<DropdownContentProps> = ({
+  dropdownSize = 'l',
+  disabled,
+  isOpen,
+  searchInput,
+  options,
+  multiple = false,
+  value,
+  name,
+  ariaLabel,
+  buttons = false,
+  cancelButtonLabel,
+  proceedButtonLabel,
+  onChange,
+  setIsOpen,
+  helperText,
+  selectedOptionObjects,
+  tags,
+}) => {
+  const [selectedValue, setSelectedValue] = useState<string[]>(
+    value ? (Array.isArray(value) ? value : [value]) : []
```

Comment:
**[blocker]**

With typescript you have defined that value is either a string or undefined, so it should never be an array. This means that 

1. Currently this check is redundant, since we know what the type is 
2. Or this is typed incorrectly